### ISSUE TITLE
Home Page 'Hero' Style for Primary Background Section Title

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1369]
**Block:** hero

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=304-249&t=3Tqy8PfO5UjdJHE0-0

**User story:** As a Web Liaison, I want the ability to display a visual contrast break for a new section on the home page so that users can identify groups of content easily. 

**Before starting the development work for this:** 
1. Review code base to see where the hero (text) is used to ensure no regression issues with this update

**Requirements:** 
1. Remove hardcoded background image (mosaic)
2. Add primary color background
3. Title displays in white text with the hero (text) style applied to the block
4. Title text displays as Heading XXL (H1) style with white text (need to specifically update these to use the H1 global style)
5. Title text should have H1 tag for hero (text) style and H2 tag for the Hero (text, homepage) style
6. Banner and title are readable by screen reader in logical page order with appropriate SEO heading tags

**Acceptance Criteria:** 
1. Given I am viewing a page with a hero banner (text), when the component block renders, then the block displays with a primary background color that spans full-width with a white title. 
2. Given I am viewing a hero banner (text) styled block, when the component renders, then the title is styled in white font in the H1 size.
3. Given I am viewing a page with a hero banner (text, homepage), when using a screen reader, then this styled hero will read out as the H1 on the page. 
4. Given I am viewing a page with a hero banner (text), when using a screen reader, then this styled hero will read out as the H2 on the page.

**Comment by aaronstewart-dev** [comment]
@beccasuchanek Latest hero adjustments are on develop. Assigning QA label again.

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

**Test Page URL:** https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/home-page-components

**Status of ACs**
1. ✅ Given I am viewing a page with a hero banner (text), when the component block renders, then the block displays with a primary background color that spans full-width with a white title.

<img width="1621" height="780" alt="Image" src="https://github.com/user-attachments/assets/c6b07121-48d8-4ed9-8a19-10857bec6d8c" />

2. ✅ Given I am viewing a hero banner (text) styled block, when the component renders, then the title is styled in white font in the H1 size. 

<img width="1643" height="348" alt="Image" src="https://github.com/user-attachments/assets/8fafe521-4cad-4d0f-a36e-b90497b045a1" />

3. ✅ Given I am viewing a page with a hero banner (text, homepage), when using a screen reader, then this styled hero will read out as the H1 on the page. 

https://github.com/user-attachments/assets/622d6cc3-3100-4191-a31f-c51877535d33

4. ✅ Given I am viewing a page with a hero banner (text), when using a screen reader, then this styled hero will read out as the H2 on the page. 

<img width="1513" height="742" alt="Image" src="https://github.com/user-attachments/assets/c016a503-07e6-42c9-a0a9-abf6785cb7ee" />

Previous comment has been resolved, see proof of testing below:
- **ISSUE 1:** 
<img width="1494" height="743" alt="Image" src="https://github.com/user-attachments/assets/3e0efbf9-ee7a-40d8-9b2b-bace03600633" />
